src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -635,6 +754,23 @@ private class FastqDemultiplexer(val sampleInfos: Seq[SampleInfo],
     case n => throw new IllegalArgumentException(s"$n template reads defined. Can't process > 2 template reads.")
   }
 
+  /** Gets the two items [[T]] with the largest associated values ([[Int]]). */
+  private def getTopTwoByMaximum[T](elems: TraversableOnce[(T, Int)]): (Option[(T, Int)], Option[(T, Int)]) = {
+    val startBest: Option[(T, Int)] = None
```

Comment:
I'm assuming you're doing this because in-lining `None` doesn't infer the type(s) correctly?  I think it would be clearer if you did:

```scala
val empty: Option[(T, Int)] = None
elems.foldLeft((empty, empty)) { case ....
```